src/RuleMailCollector.php

```diff
@@ -110,6 +110,12 @@ public function getCriterias()
         $criterias['x-uce-status']['table']             = '';
         $criterias['x-uce-status']['type']              = 'text';
 
+        $criterias['_headers'] = [
+            'name'  => __('Full email headers'),
```

Comment:
Maybe this would be more precise:
```suggestion
            'name'  => __('Entire email header string'),
```

@cconard96 What would be the most relevant text here?

Reply:
Just `Headers` would be fine IMO.

If headers contains XXX then ...